src/main/java/eu/dissco/core/translator/terms/BaseDigitalObjectDirector.java

```diff
@@ -545,15 +565,13 @@ private List<Event> assembleEventTerms(JsonNode data, boolean dwc) {
             termMapper.retrieveTerm(new DegreeOfEstablishment(), data, dwc))
         .withDwcGeoreferenceVerificationStatus(
             termMapper.retrieveTerm(new GeoreferenceVerificationStatus(), data, dwc))
-        .withDwcOccurrenceStatus(
-            retrieveEnum(new OccurrenceStatus(), data, dwc, DwcOccurrenceStatus.class))
-        .withDwcOccurrenceRemarks(termMapper.retrieveTerm(new OccurrenceRemarks(), data, dwc))
+        .withDwcEventRemarks(termMapper.retrieveTerm(new EventRemarks(), data, dwc))
         .withDwcSampleSizeUnit(termMapper.retrieveTerm(new SampleSizeUnit(), data, dwc))
         .withDwcSampleSizeValue(parseToDouble(new SampleSizeValue(), data, dwc))
         .withDwcCaste(termMapper.retrieveTerm(new Caste(), data, dwc))
         .withDwcVitality(termMapper.retrieveTerm(new Vitality(), data, dwc))
-        .withOdsLocation(location)
-        .withOdsHasAssertion(assertions);
+        .withOdsHasLocation(location)
+        .withOdsHasAssertions(assertions);
     return List.of(event);
   }
 
```

Comment:
Yeah we talked about this and I checked. Generally I would say not. It is difficult with boolean. What does null, or false mean. So generally I would say this is null (not included in object) if we cannot find the field in the data. `false` is then there if the actual value in the data says it isn't. So for example the `isMarkedAsType` can be true when it is a type, `false` when it is not a type, or not present when we just don't know. If we fix this at `false` then we won't be able to indicate that we just don't know.
For media it is a bit different as it will always be true or false but we catch that in the term class which will always return one or the other

Reply:
Booleans in data are tricky. It's a binary type that needs to represent 3 different options. :upside_down_face: No way around it unless we introduce a new standard

Reply:
I know, I know. Talked about this with MIDS as well (as it is used for the calculation) but we didn't got much movement in it and it is technically possible this way